x-pack/plugin/deprecation/qa/rest/src/javaRestTest/java/org/elasticsearch/xpack/deprecation/DeprecationHttpIT.java

```diff
@@ -320,6 +321,131 @@ public void testDeprecationMessagesCanBeIndexed() throws Exception {
         }
     }
 
+    /**
+     * Check that compatible api log messages can be recorded to an index
+     */
+    public void testCompatibleMessagesCanBeIndexed() throws Exception {
```

Comment:
should we also test the HTTP response warning ?

Reply:
nevermind on this comment ... since it is the same functionality as other deprecation logger, that is already tested.